go/private/context.bzl

```diff
@@ -643,8 +644,12 @@ def _cgo_context_data_impl(ctx):
     # toolchain (to be inputs into actions that need it).
     # ctx.files._cc_toolchain won't work when cc toolchain resolution
     # is switched on.
-    cc_toolchain = find_cpp_toolchain(ctx)
-    if cc_toolchain.compiler in _UNSUPPORTED_C_COMPILERS:
+    find_cpp_toolchain_kwargs = {}
+    if bazel_features.cc.find_cpp_toolchain_has_mandatory_param:
+        find_cpp_toolchain_kwargs["mandatory"] = False
```

Comment:
I would like this better if we just duplicated the call instead of using `kwargs`.